pom.xml

```diff
@@ -114,7 +114,7 @@
         <yarn.version>1.22.4</yarn.version>
         
         <!-- Quarkus Version -->
-        <quarkus.version>1.5.0.Final</quarkus.version>
+        <quarkus.version>1.5.1.Final</quarkus.version>
```

Comment:
The last upstream Quarkus [version](https://quarkus.io/blog/quarkus-1-5-2-final-released/) is `1.5.2.` In that blog, they state that it is safe to upgrade from `1.5.1` to `1.5.2`.